### PR TITLE
chore: add issue template

### PR DESCRIPTION
**Pull request by yordis**
Trying to help users on what to do by leveraging the Issue Templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      required: true
+
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current behavior
+      description: >
+        How can we see what you're seeing? Specific is terrific. Logs?
+        Screenshots? Include code samples, errors and stacktraces if
+        appropriate. Yes, please.
```
Comment (josevalim) [inline .github/ISSUE_TEMPLATE/bug.yml:45]:
```suggestion
        How can we reproduce what you're seeing? Include code
        samples, errors and stacktraces if appropriate.
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    attributes:
+      label: Ecto Versions
+      description: Use `mix deps` to find the dependency versions.
+    validations:
+      required: true
+
+  - type: textarea
+    id: db-adapter-version
+    attributes:
+      label: Database Adapter and Versions
```
Comment (josevalim) [inline .github/ISSUE_TEMPLATE/bug.yml:33]:
```suggestion
      label: Database Adapter and Versions (postgrex, myxql, etc) 
```

Or maybe a placeholder is better?

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by yordis** [comment]
@josevalim I can do this for `ecto_sql` as well, and elixir if you would like me to

**Comment by josevalim** [comment]
Yes, pls!

**Pull request merged**